_resources/getting-started-with-distributed-organizing.md

```diff
@@ -1,9 +1,13 @@
 ---
 layout: resource
 title: Getting Started with Distributed Organizing
+resource_url: /foo.pdf
```

Comment:
This URL is the placeholder you mentioned on the PR details?

Reply:
Yes.